CMakeLists.txt

```diff
@@ -20,6 +20,12 @@ set(hpxmp_headers
         src/kmp_atomic.h
         src/loop_schedule.h)
 
+set(HPXMP_WITH_TRACE ON CACHE BOOL "display traces in debug mode")
+
+if(HPXMP_WITH_TRACE)
+    add_definitions(-DHPXMP_HAVE_TRACE=ON)
+endif()
```

Comment:
This will lead to compilation errors as `ON` is undefined in C++ land. Use `add_definitions(-DHPXMP_HAVE_TRACE)` or `add_definitions(-DHPXMP_HAVE_TRACE=1)` instead.

Reply:
... it probably compiled fine as you never use the value of this pp constant in the code below.